prac_05/state_names.py

```diff
@@ -20,3 +20,4 @@ def main():
         print("{:3} is {:}".format(state, STATE_NAMES[state]))
 
 main()
+
```

Comment:
Dictionary literals should be formatted with no space before, and one space after, the colon